pkg/solana/monitor/balance.go

```diff
@@ -62,14 +62,14 @@ func (b *balanceMonitor) Name() string {
 }
 
 func (b *balanceMonitor) Start(context.Context) error {
-	return b.StartOnce("SolanaBalanceMonitor", func() error {
+	return b.StartOnce("BalanceMonitor", func() error {
```

Comment:
just double checking - is there other indication that this is a solana balance monitor? (with and without loop)